packages/cms-lib/personalAccessKey.js

```diff
@@ -33,9 +33,14 @@ async function getAccessToken(personalAccessKey, env = 'PROD') {
     response = await fetchAccessToken(personalAccessKey, env);
   } catch (e) {
     if (e.response) {
-      throw new HubSpotAuthError(
-        `Error while retrieving new access token: ${e.response.body.message}`
-      );
+      const errorOutput = `Error while retrieving new access token: ${e.response.body.message}.`;
```

Comment:
We have some conditionals elsewhere that indicate you could get a reference error here.
https://github.com/HubSpot/hubspot-cms-tools/search?q=response.body&unscoped_q=response.body